src/box/lua/schema.lua

```diff
@@ -3500,12 +3529,29 @@ local function revoke(uid, name, privilege, object_type, object_name, options)
     end
     local privilege_hex = privilege_check(privilege, object_type)
     options = options or {}
+    if options.origin == nil then
+        options.origin = ''
+    elseif type(options.origin) ~= 'string' or #options.origin == 0 then
```

Comment:
I think it is OK to allow an explicit `origin = ''`. A user may see it in the `_priv` space or in the `.info()` endpoint. If would be logical to expect that it is accepted.

Reply:
BTW, I would use `DEFAULT_ORIGIN = ''` constant or something of this kind across the code.

Reply:
Thanks, fixed.